### ISSUE TITLE
addict.generate.configs fails on project addict_demo

### ISSUE THREAD

**Issue by smpallen99** [trenpixster/addict#78]
I get the following error:

```
➜  addict_demo git:(master) ✗ mix addict.generate.configs
[x] Please remove the existing Addict configuration before generating a new one
[o] Done!
```

It looks like `addict_config_already_exists?` needs a little more logic.

**Comment by trenpixster** [comment]
Closed via #79

**Comment by smpallen99** [comment]
Another quick note. It will also fail if config entry is commented out. This might be a real scenario wanting to compare results from selecting different options. I'll leave it up to u as to whether u believe it warrants an issue. I would have included it in my PR but it's not a trivial regex for me.
